schema_editor/app/scripts/views/recordtype/detail-add-edit-partial.html

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
Yeah, I thought about that too. I could see these changing down the line to be more different, so I just left it as two separate views. But I don't feel strongly about it.